src/lib/InstantSearch.ts

```diff
@@ -111,7 +111,7 @@ export type InstantSearchOptions<TRouteState = UiState> = {
  * created using the `instantsearch` factory function.
  * It emits the 'render' event every time a search is done
  */
-class InstantSearch extends EventEmitter {
+class InstantSearch extends EventEmitter.EventEmitter {
```

Comment:
what made this change necessary? It doesn't seem like you've updated that dependency

Reply:
Since the lock file was regenerated, the internal `events` dependency is now `3.1.0`:

https://github.com/algolia/instantsearch.js/blob/3b831507df3a5ba70b1db3952c45701551ceab86/yarn.lock#L6064-L6067

We probably want to revert this change.

Reply:
hmm, IIRC we want events@2, not events@3, there's a quite big difference in size, and the helper is using @2